doc/library/tensor/signal/downsample.txt

```diff
@@ -1,17 +0,0 @@
-.. _libdoc_tensor_signal_downsample:
-
-======================================================
-:mod:`downsample` -- Down-Sampling
-======================================================
-
```

Comment:
I would keep that file but put a reference to the new version. This way old link won't completly break.